t/04-scheduler.t

```diff
@@ -355,7 +370,6 @@ is_deeply($grabbed->{settings}, $job_ref->{settings}, "settings correct");
 my $job3_id = $job->id;
 my $job_id  = $grabbed->{id};
 
-sleep 1;
```

Comment:
you are deleting sleeps in code. I love you for this already! :kissing:  :smirk: